lib/RequestRouter.js

```diff
@@ -514,8 +520,8 @@ RequestRouter.prototype.addPropertyError = function(property, errorCode) {
         });
     }
 
-    if (this._request.getMethod() == Request.METHOD_POST && this.isCollection()) {
-        //remove the appended items@ if posting to a collection
+    if (!!~[Request.METHOD_PATCH, Request.METHOD_POST, Request.METHOD_PUT, Request.METHOD_DELETE].indexOf(this.getMethod()) && this.isCollection()) {
```

Comment:
When patching a collection, we're expecting to change the properties of the collection as a whole. Not individual items in the collection. I mean, which item would be modified? If I `PATCH /users`, which user gets edited? That wouldn't make sense..